source/3d-secure-2.html.erb

```diff
@@ -0,0 +1,132 @@
+      <p class="govuk-body">Services that take payments from the EU, Norway, Iceland and Liechtenstein (the European Economic Area, EEA), need to implement SCA by December 2020. Services that only take payments from within the UK need to implement SCA by September 2021.</p>
+      <p class="govuk-body">3D Secure 2 (3DS2) adds an extra layer of identity confirmation before the user pays so that transactions meet the SCA rules.</p>
+      <p class="govuk-body">3DS2 replaces 3D Secure (3DS). 3DS transactions offer a similar level of protection for users and are compliant with SCA requirements, but introduce more friction to the users’ journey. 3DS will also be phased out, though the date for this has not been confirmed.</p>
+
+      <h2 class="govuk-heading-m">What 3D Secure 2 means for users</h2>
+      <p class="govuk-body">3DS2 will ask users for 2 out of 3 pieces of information to complete their transaction:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>something the user is, for example their fingerprint</li>
+        <li>something the user has, for example their phone</li>
+        <li>something the user knows, for example a password</li>
+      </ul>
+
+      <p class="govuk-body">Transactions processed through digital wallets (such as Apple Pay or Google Pay) meet the 2 factor authentication requirements.</p>
+
+      <p class="govuk-body">3DS2 will not apply to:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>any transactions where either the merchant or payer is outside the EEA</li>
+        <li>merchant initiated payments, for example subscriptions</li>
+        <li>telephone payments</li>
+      </ul>
+
+      <p class="govuk-body">Some payments may be exempted from 3DS2:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>low value transactions, for example for those under £30</li>
+        <li>low risk transactions when the Payment Service Provider (PSP) has low fraud levels across all its platform</li>
+        <li>corporate payments (unless the corporate card is in the name of an individual)</li>
+      </ul>
+
+      <p class="govuk-body">Which exemptions are applied will depend on your Payment Service Provider (PSP).</p>
+
+      <h2 class="govuk-heading-m">What 3D Secure 2 means for services</h2>
+      <p class="govuk-body">You may see a reduction in fraud on your service because users need to provide additional authentication before making a payment.</p>
+      <p class="govuk-body">Services are not liable for fraudulent or unrecognised transactions if they are processed by 3DS2. If a user disputes a 3DS2 transaction (for example via a chargeback) on the basis that it is fraudulent or unrecognised, it is the responsibility of the bank to refund the customer if required, because the bank approved 3DS2 transactions.</p>
```

Comment:
Replace "via" with "using" as more accessible and plain english for non-native speakers